src/ui/win32/bindings/MemoryViewerControlBinding.cpp

```diff
@@ -231,6 +232,56 @@ bool MemoryViewerControlBinding::HandleNavigation(UINT nChar)
             }
             return true;
 
+        case 'C':
+            if (bControlHeld)
+            {
+                const auto& pEmulatorContext = ra::services::ServiceLocator::Get<ra::data::context::EmulatorContext>();
+                const auto iValue = pEmulatorContext.ReadMemory(m_pViewModel.GetAddress(), m_pViewModel.GetSize());
+                std::wstring sValue = ra::data::MemSizeFormat(iValue, m_pViewModel.GetSize(), MemFormat::Hex);
+
+                ra::services::ServiceLocator::Get<ra::services::IClipboard>().SetText(ra::Widen(sValue));
+            }
+            return true;
+
+        case 'V':
+            if (bControlHeld)
+            {
+                auto nAddress = m_pViewModel.GetAddress();
+                const auto& pEmulatorContext = ra::services::ServiceLocator::Get<ra::data::context::EmulatorContext>();
+                std::wstring sClipboardText = ra::services::ServiceLocator::Get<ra::services::IClipboard>().GetText();
+                auto n = m_pViewModel.GetSize();
```

Comment:
`n` is not a very descriptive variable name, and I can't tell if it's even being used anywhere.